Documentation/artifact-format.md

```diff
@@ -154,14 +230,36 @@ the two spaces.
 manifest.sig
 ----
 
-Format: TBD
+Format: base64 encoded ecdsa or rsa signature
 Version: Exists only in version 2 and later
 
 File containing the signature of `manifest`.
 
 It is legal for an artifact not to have signature file.
 
 
+manifest-augment
+----
+
+Format: text
+Version: Exists only in version 3 and later
+
+Contains the file checksums, formatted exactly like below:
```

Comment:
"Contains ~~the~~ file checksums". Otherwise it sounds like it contains all checksums.

Reply:
Fixed!